test/server.ts

```diff
@@ -268,18 +269,20 @@ async function testParallelServerStart (options: {
         ]),
       createDeferred<void>()]
     serverProcessList.push(item)
-    item[0].on('exit', (code: number) => {
+    byline(item[0].stderr).on('data', (line: Buffer) => console.error(`${item[0].pid}: ${line}`))
+    byline(item[0].stdout).on('data', (line: Buffer) => console.log(`${item[0].pid}: ${line}`))
```

Comment:
It is not good to use `console.log` in tests because we use the tap format to output results of tests. And outputting via console might make the format incompatible with different test reporters. You could use [t.comment(...)](https://www.npmjs.com/package/tape#tcommentmessage)